### PR TITLE
AddingImages: enable to set beta param if use_data is true

### PR DESCRIPTION
**Pull request by k-okada**
addweight

### CODE REVIEW

File: src/nodelet/adding_images_nodelet.cpp [hunk 1]
```diff
@@ -171,9 +171,9 @@ namespace adding_images {
       // Work on the image.
       try {
         cv::Mat image1 =
-          cv_bridge::toCvShare(image_msg1, sensor_msgs::image_encodings::BGR8)->image;
+          cv_bridge::toCvShare(image_msg1, image_msg1->encoding)->image;
         cv::Mat image2 =
-          cv_bridge::toCvShare(image_msg2, sensor_msgs::image_encodings::BGR8)->image;
+          cv_bridge::toCvShare(image_msg2, image_msg2->encoding)->image;
```
Comment (k-okada) [inline src/nodelet/adding_images_nodelet.cpp]: we need to assume two input are same encoding.

**Comment by wkentaro** [comment]
Need https://github.com/k-okada/opencv_apps/pull/1

**Comment by wkentaro** [comment]
LGTM

**Comment by k-okada** [comment]
@wkentaro please check if 3ed06e9 works.

**Comment by wkentaro** [comment]
OK

**Comment by wkentaro** [comment]
We need https://github.com/k-okada/opencv_apps/pull/2

**Comment by wkentaro** [comment]
```
[0m/home/travis/catkin_ws/src/opencv_apps/src/nodelet/adding_images_nodelet.cpp: In member function 'void adding_images::AddingImagesNodelet::do_work(const ConstPtr&, const ConstPtr&, std::string)': [0m[0m
/home/travis/catkin_ws/src/opencv_apps/src/nodelet/adding_images_nodelet.cpp:191:11: error: 'CvtColorForDisplayOptions' is not a member of 'cv_bridge' [0m[0m
/home/travis/catkin_ws/src/opencv_apps/src/nodelet/adding_images_nodelet.cpp:191:48: error: expected ';' before 'options' [0m[0m
/home/travis/catkin_ws/src/opencv_apps/src/nodelet/adding_images_nodelet.cpp:195:13: error: 'options' was not declared in this scope [0m[0m
/home/travis/catkin_ws/src/opencv_apps/src/nodelet/adding_images_nodelet.cpp:197:36: error: 'cvtColorForDisplay' is not a member of 'cv_bridge' [0m[0m
/home/travis/catkin_ws/src/opencv_apps/src/nodelet/adding_images_nodelet.cpp:197:104: error: 'options' was not declared in this scope [0m[0m
make[2]: *** [CMakeFiles/opencv_apps.dir/src/nodelet/adding_images_nodelet.cpp.o] Error 1 [0m[0m
make[1]: *** [CMakeFiles/opencv_apps.dir/all] Error 2 [0m[0m
```

**Comment by wkentaro** [comment]
cvtColorForDisplay is also unsupported..

**Pull request merged**